Standards/scs-0302-v1-domain-manager-role.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Domain Manager configuration for Keystone
+type: Standard
+status: Draft
+track: IAM
+---
+
+## Introduction
+
+SCS Clouds should provide a way to grant Domain Manager rights to SCS Customers which provides IAM self-service capabilities within an OpenStack domain.
+This is not properly implemented in the default OpenStack configuration and requires specific adjustments to the Keystone identity management configuration.
+To avoid conflict with the unscoped `admin` role in OpenStack we want to refer to this new role as "Domain Manager" (`domain-manager`).
+
+### Glossary
```

Comment:
I like your glossary. :-) We should add `Glossary` as additional mandatory section in our [standard for standards](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md).